pages/common/7zr.md

```diff
@@ -1,17 +1,8 @@
 # 7zr
 
-> A file archiver with high compression ratio.
-> A standalone version of `7z` that only supports .7z files.
-> More information: <https://www.7-zip.org/>.
+> This command is an alias of 7z.
```

Comment:
This is _not_ true. They are _not_ aliases, and it's misleading to represent them as such. `7za` and `7zr` are completely separate binaries. The difference between them is that `7za` supports fewer formats and is a static binary, where `7z` uses plugins to support other file formats.

`7zr` in particular supports _only_ 7z files and hence the `-t` isn't actually useful, whereas the `7za` and `7z` have multiple archive formats they support.

Reply:
I was afraid of something like that.  It was really just an experiment :). I don't mind changing them at all.  Only thing is, it'll need to be updated in #5437.  cc @marchersimon

Reply:
No worries. The alias syntax as a whole is great idea, but it does not work here because these commands - although similar - are not aliases.